bittensor/commands/utils.py

```diff
@@ -15,6 +15,7 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+from munch import Munch, munchify
 import sys
 import os
 import bittensor
```

Comment:
```suggestion
import sys
import os
from munch import Munch, munchify
import bittensor
```
nit: move munch after os,sys

Reply:
Good catch! I forgot to optimize the import in this file.